backstage-plugin/plugins/open-dora/src/components/DashboardComponent/DashboardComponent.tsx

```diff
@@ -85,6 +95,10 @@ export const DashboardComponent = ({
           });
         },
       );
+
+    groupDataService.retrieveBenchmarkData({ type: 'df' }).then(response => {
+      setDfOverview(response.key);
+    });
```

Comment:
This should have an error handler if there is a network error, error response or an issue with config. For now we can just log the error to the console but we can display the error similar to the chart errors in another PR.
```suggestion
     },
      // eslint-disable-next-line no-console
      (error: Error) => console.error(error),
    );
```